src/stories/Library/disclosure/disclosure.scss

```diff
@@ -19,14 +58,26 @@
     border-radius: 20px;
     display: flex;
     justify-content: center;
+    margin-right: $s-md;
+
+    @include breakpoint-s {
+      margin-left: -8px;
```

Comment:
I wonder if we can use a variable here with an expression e.g. `-#{$s-sm}`.

Reply:
Indeed we can do that! Thanks! :)